content/applications/sales/point_of_sale/restaurant/floors_tables.rst

```diff
@@ -2,91 +2,106 @@
 Floors and tables management
 ============================
 
-Add a floor
-===========
-
-Once your *Point of Sale* has been configured, select *Table
-Management* under :menuselection:`Point of Sale --> Configuration -->
-Point of Sale`. Then, click on *Floors* to create and name your floor and tables.
+Floor plan key
```

Comment:
I see you haven't applied the changes I suggested for this last section, let me know if it's voluntary and you don't want to apply them or maybe you missed them.

Reply:
I wasn't sure about the suggestion, so I kept it for later and forgot about it at some point. Now that I visualize the whole page, Floor plan would work, I'm applying it now